src/zrc/api/serializers.py

```diff
@@ -622,3 +622,26 @@ class Meta:
                 ],
             }
         }
+
+
+class ZaakAuditTrailSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = AuditTrail
+        fields = (
+            'uuid',
+            'bron',
+            # 'applicatieId',
+            # 'applicatieWeergave',
+            # 'gebruikersId',
+            # 'gebruikersWeergave',
+            'actie',
+            'actieWeergave',
```

Comment:
We can store the (above define) DjangoChoices representation of the constant for `actieWeergave` by default.